concent_api/core/tests/test_auth_accept_or_reject_results.py

```diff
@@ -443,7 +443,7 @@ def test_requestor_sends_subtask_results_accepted_and_concent_should_return_it_t
             timestamp="2018-02-05 10:00:43",
             subtask_results_accepted=self._get_deserialized_subtask_results_accepted(
                 timestamp="2018-02-05 10:00:43",
-                payment_ts="2018-02-05 10:00:44",
+                payment_ts="2018-02-05 9:00:44",
```

Comment:
In tests there is lot of changes like this - why is it changed and why exactly one hour?

Reply:
Answered in https://github.com/golemfactory/concent/pull/1050#discussion_r245933221